src/index.html

```diff
@@ -17,11 +17,317 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.css"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo"><img src="./images/logo.png" alt="logo"></div>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li><a href="#" class="header__nav-link is-active">Apple</a></li>
+          <li><a href="#" class="header__nav-link">Samsung</a></li>
+          <li><a href="#" class="header__nav-link">Smartphones</a></li>
+          <li><a href="#" class="header__nav-link" data-qa="nav-hover">Laptops & Computers</a></li>
+          <li><a href="#" class="header__nav-link">Gadgets</a></li>
+          <li><a href="#" class="header__nav-link">Tablets</a></li>
+          <li><a href="#" class="header__nav-link">Photo</a></li>
+          <li><a href="#" class="header__nav-link">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+
+      <article
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="Product Image"
+        class="card__image"
+      />
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="stars stars--4">
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
+
+      <p class="card__reviews">Reviews: 5</p>
+      <p class="card__price">
+        Price:
+        <span class="card__price-value">$2,199</span>
+      </p>
+      <a
+        href="#"
+        class="card__buy"
+        data-qa="hover"
```

Comment:
The `data-qa="card-hover"` attribute is missing. According to the task requirements, this attribute should be present to facilitate testing of the hover effect.